src/packages/cli/src/process-name.ts

```diff
@@ -0,0 +1,85 @@
+function pick(source: string[]) {
+  const partIndex = Math.floor(Math.random() * source.length);
+  return source[partIndex];
+}
+/**
+ * Generates a random name to assign to an instance of Ganache. The name is
+ * generated from an adjective, a flavor and a type of desert, in the form of
+ * `<adjective>_<flavor>_<type>`, eg., `salted_caramel_ganache`.
+ */
+export default function createInstanceName() {
+  const name = [adjectives, flavors, kinds].map(pick).join("_");
```

Comment:
This current code doesn't bother me, so feel free to ignore this, but the below suggestion avoids the creation of an array and two loops. :-p

```suggestion
  const name = `${pick(adjectives)}_${pick(flavors)}_${pick(kinds)}`;
```

Reply:
Yeah this is probably better.